python/PrintOldLogs.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime, timedelta
+import os
+
+def PutColor(a,color):
+  return '${'+color+'}'+a+'${Color_Off}'
+
+USER = os.environ['USER']
+if os.path.exists(os.environ['SKFlat_WD']+'/python/UserInfo_'+USER+'.py'):
+  exec('from UserInfo_'+USER+' import *')
+else:
+  print("No UserInfo file")
+  exit(1)
+LogRemoval = int(UserInfo['LogRemoval'])
```

Comment:
Could you add some code to handle the case when there is no 'LogRemoval' in UserInfo file to increase compatibility with existing UserInfo files?
For example,
```python
if 'LogRemoval' not in UserInfo:
    LogRemoval=-1 # or a large value?
else:
    LogRemoval = int(UserInfo['LogRemoval'])
```